data-types/collections/spec/hashes_spec.rb

```diff
@@ -77,48 +88,48 @@
     expect(books).to eq(expected)
   end
 
-  xit 'test 8' do
+  it 'test 8' do
     books = {
       "John Steinbeck" => "Grapes of Wrath",
       "Harper Lee" => "To Kill a Mockingbird"
     }
     # Using the books hash defined above,
     # change the value associated with
     # "John Steinbeck" to "Of Mice and Men"
-    ______
+    books= {"John Steinbeck" => "Of Mice and Men" , "Harper Lee" => "To Kill a Mockingbird" }
```

Comment:
Nope :)

you need to change the value of `John Steinback` and you are just declaring a new hash which is not what question is asking.